Parameters.xml

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
please use lowercase for file names. Ex:
`parameterizedSuite.xml`
`allTestSuite.xml`
don't use spaces and dashes for file names